src/components/marker-chart/Canvas.js

```diff
@@ -43,6 +43,7 @@ type Props = {
   markerTimingRows: MarkerTimingRows,
   rowHeight: CssPixels,
   markers: TracingMarker[],
+  getTooltipContents: TracingMarker => ReactElement<any>,
```

Comment:
maybe use `React.Node` as return value (this means: anything than can be rendered by react).
`React.Element<any>` is part of `React.Node` so Flow will happily accept the passed function.

Reply:
Then I also need to adjust `getHoveredMarkerInfo`, but the comment above it explicitly says that that's ok, so I'm going to do that :)